test/spectests/README.md

```diff
@@ -33,7 +33,10 @@ In order to prepare the tests, run the following command for each file:
 $ wast2json <file.wast> -o <file.json>
 ```
 
-For ease of use, we have placed the JSON files of the [spec tests] v1.1 release here: https://github.com/wasmx/wasm-spec/tree/vanilla-json
+For ease of use, we have placed the JSON files of the spec tests [w3c-v1.0 branch] here:
+- [Used before Fizzy 0.4](https://github.com/wasmx/wasm-spec/releases/tag/w3c-1.0-jsontests-20200313)
```

Comment:
Do we need to mention the branch `vanilla-json` here too?

Reply:
No.